example_site/src/components/App.css

```diff
@@ -35,38 +36,30 @@ main {
   grid-template-columns: repeat(2, 1fr);
 }
 
-.webhook-testing-container {
-  display: grid;
-  grid-template-columns: repeat(2, 1fr);
-}
-
 .test-form-grid {
   display: grid;
   grid-template-columns: repeat(2, 1fr);
 }
 
-.panel {
-  padding: 24px;
-  margin-bottom: 8px;
-  border-radius: 4px;
-  background-color: white;
-  margin: 12px;
-  overflow: auto;
-}
-
 .form-label {
   display: block;
   font-weight: bold;
   margin: 12px 0 4px;
 }
 
+.center {
```

Comment:
You can just add these styles to the `main` section above instead of defining a new class